ssg/ansible.py

```diff
@@ -100,6 +100,8 @@ def update_tags_from_config(self, to_update):
             tags.append("{0}_complexity".format(self.config["complexity"]))
         if "disruption" in self.config:
             tags.append("{0}_disruption".format(self.config["disruption"]))
+        if "reboot" in self.config:
+            tags.append("{0}_reboot".format(self.config["reboot"]))
```

Comment:
`false_reboot` / `true_reboot` are somehow confusing, what about `reboot_required` / `no_reboot_needed`?

Reply:
yes, thank you for the suggestion

Reply:
@matejak changed